src/compiler.ts

```diff
@@ -1,11 +1,49 @@
 #!/usr/bin/env node
-import { readFileSync, readdirSync, statSync } from 'fs';
+import { readFileSync, readdirSync, statSync, existsSync } from 'fs';
 import { join, basename } from 'path';
 import asc from 'assemblyscript/dist/asc.js';
 import yargs from 'yargs';
 import { hideBin } from 'yargs/helpers';
+import { APIOptions } from 'assemblyscript/dist/asc.js';
 
-async function compile(argv: string[], options: object = {}): Promise<boolean> {
+const asconfigPath = './asconfig.json';
+
+type CompilationOptions = APIOptions & {
```

Comment:
Why aren't you using compiler [Option](https://github.com/AssemblyScript/assemblyscript/blob/main/src/compiler.ts#L231)?